### PR TITLE
Custom Shop

### PR DESCRIPTION
**Pull request by Chronophylos**
Feel free to contribute.
I copied the code and applied semi-standard
I don't know if this works and I'm sure I missed something

### CODE REVIEW

**Comment by Chronophylos** [comment]
```Shell
semistandard: Semicolons For All! (https://github.com/Flet/semistandard)
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:56: Unnecessary escape character: \-.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:58: Unnecessary escape character: \[.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:62: Unnecessary escape character: \/.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:64: Unnecessary escape character: \{.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:66: Unnecessary escape character: \}.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:68: Unnecessary escape character: \(.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:70: Unnecessary escape character: \).
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:72: Unnecessary escape character: \*.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:74: Unnecessary escape character: \+.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:76: Unnecessary escape character: \?.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:78: Unnecessary escape character: \..
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:82: Unnecessary escape character: \^.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:84: Unnecessary escape character: \$.
  /home/travis/build/OpenAngelArena/oaa/content/panorama/scripts/panorama_shop.js:28:86: Unnecessary escape character: \|.

npm ERR! Test failed.  See above for more details.
```
funny

File: game/scripts/vscripts/components/shop/shop.lua [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+  local hero = PlayerResource:GetSelectedHeroEntity(playerID)
+  local team = PlayerResource:GetTeam(playerID)
+  if Duel:IsDuelOngoing() then
+    Containers:DisplayError(playerID, "#dota_hud_error_cant_purchase_duel_ongoing")
+    return
+  end
+  if unit:IsIllusion() or not unit:HasInventory() then
+    unit = hero
+  end
+  local isInShop = unit:HasModifier("modifier_fountain_aura_arena")
```
Comment (Trildar) [inline game/scripts/vscripts/components/shop/shop.lua]: Need to change this to the built-in fountain modifier I think.

**Comment by Chronophylos** [comment]
Thanks.
Yeah sure open a PR on my branch.

**Comment by warpdragon** [comment]
diving in now...

**Comment by warpdragon** [comment]
curious. what happens when you attempt to run this OOTB?

**Comment by Chronophylos** [comment]
ATM it doesn't work.
What I found is that nothing happens. I don't know about what happens in the basckground

**Comment by chrisinajar** [comment]
Closing dead PRs

**Comment by Chronophylos** [comment]
I deleted the branch too.

**Pull request closed without merging**